packages/react-charting/src/components/AreaChart/AreaChart.base.tsx

```diff
@@ -187,12 +193,16 @@ export class AreaChartBase extends React.Component<IAreaChartProps, IAreaChartSt
           chartType={ChartTypes.AreaChart}
           calloutProps={calloutProps}
           legendBars={legends}
+          createYAxis={createNumericYAxis}
           isCalloutForStack
           xAxisType={isXAxisDateType ? XAxisTypes.DateAxis : XAxisTypes.NumericAxis}
           tickParams={tickParams}
           maxOfYVal={stackedInfo.maxOfYVal}
           getGraphData={this._getGraphData}
+          getDomainNRangeValues={this._getDomainNRangeValues}
+          createStringYAxis={createStringYAxisForOtherCharts}
```

Comment:
this function is still not renamed

Reply:
renamed createStringYAxisForOtherCharts to createStringYAxis